webapp/sources/rudder/rudder-web/src/main/scala/com/normation/rudder/web/services/AsyncComplianceService.scala

```diff
@@ -200,4 +200,17 @@ class AsyncComplianceService (
     compliance(kind,tableId)
 
   }
+
+  def nodeCompliance(nodeId: NodeId): Box[ComplianceLevel] = {
+    val node = new NodeCompliance(Set(nodeId), Set())
+    node.computeCompliance match {
+      case Full(contentMap) =>
+        contentMap.values.head match {
```

Comment:
head throws exception too, especially if this is empty

You shoud should do 

<pre>contentMap.get(nodeId) match 
  case Some(Some(compiance)) => Full(Compliance)
  case _ =>
</pre>